pages/common/deepseek.md

```diff
@@ -0,0 +1,28 @@
+# deepseek
+
+> A command-line interface for interacting with DeepSeek language models.
+> More information: <https://github.com/deepseek-ai>.
```

Comment:
Change this link to something that directly instructs on how to use the command.